source.go

```diff
@@ -3,12 +3,13 @@ package omise
 // Source represents Omise's source object.
 // See https://www.omise.co/source-api for more information.
 type Source struct {
```

Comment:
should there be other missing field as well? e.g. `references`, `zero_interest_installments`